scripts/templates/2.0/registrar.j2

```diff
@@ -80,3 +80,42 @@ prov_db_filename: {{ registrar.prov_db_filename }}
 # 'stream': The log output will be written to a common file in
 #           'var/log/keylime/keylime-stream.log'
 log_destination = {{ registrar.log_destination }}
+
+# In order to enable Durable Attestation, an "adapter" for a Persistent data Store 
```

Comment:
I would also add a note here and in the verifier config that this feature is experimental.

Reply:
yes